src/sql/QueryEditor/FillValueSelect.tsx

```diff
@@ -64,7 +64,7 @@ export function FillValueSelect<TQuery extends DataQuery & Record<string, any>>(
                 },
               })
             }
-            onBlur={() => props.onRunQuery()}
+            onBlur={() => props?.onRunQuery?.()}
```

Comment:
nit but `props` should be defined (at least according to the function signature), so all the calls should be `props.onRunQuery?`